tests/robot-tests/tests/general_public/methodologies_page.robot

```diff
@@ -20,19 +20,13 @@ Validate page contents
     user waits until page contains accordion section    Pupils and schools
     user opens accordion section    Pupils and schools
 
-    user opens details dropdown    Exclusions
-    user checks page contains methodology link
-    ...    Exclusions
-    ...    Permanent and fixed-period exclusions in England
+    user checks page contains link with text and url
     ...    Pupil exclusion statistics: methodology
-    ...    %{PUBLIC_URL}/methodology/permanent-and-fixed-period-exclusions-in-england
+    ...    /methodology/permanent-and-fixed-period-exclusions-in-england
 
-    user opens details dropdown    Pupil absence
-    user checks page contains methodology link
-    ...    Pupil absence
-    ...    Pupil absence in schools in England
+    user checks page contains link with text and url
     ...    Pupil absence statistics: methodology
-    ...    %{PUBLIC_URL}/methodology/pupil-absence-in-schools-in-england
```

Comment:
Just wondering why we've gotten rid of the `PUBLIC_URL` variable here?

Reply:
I think it was because `user checks page contains methodology link` was checking the link url in a different way to `user checks page contains link with text and url`